docs/learn/learn-inflation.md

```diff
@@ -84,22 +84,22 @@ network, please see [here](https://research.web3.foundation/Polkadot/overview/to
 
 ## Ideal Staking Rate
 
-The ideal staking rate can vary between 45% to 75% based on the number of parachains that acquired a
-lease through an auction (this excludes the System parachains), based on the implementation
+The ideal staking rate can vary between 45% to 75% based on the number of parachains that occupied a
+core (this excludes the system parachains), based on the implementation
 [here](https://github.com/paritytech/polkadot-sdk/blob/cd901764a52edc04a6d22bea3a526def593ab2a7/polkadot/runtime/common/src/impls.rs#L80).
 
 Briefly, the ideal staking rate can be calculated as follows:
 
-`0.75 - auction_proportion`
```

Comment:
These are variables from the code. There is no `auction_proportion` variable in the code.

It makes sense to redo this inflation section as things have changed recently

Reply:
agree, I will update to most recent changes